src/cartservice/src/Program.cs

```diff
@@ -61,16 +61,25 @@
 builder.Services.AddOpenTelemetry()
     .ConfigureResource(appResourceBuilder)
     .WithTracing(tracerBuilder => tracerBuilder
+        .AddSource("OpenTelemetry.Demo.Cart.Source")
         .AddRedisInstrumentation(
             options => options.SetVerboseDatabaseStatements = true)
         .AddAspNetCoreInstrumentation()
         .AddGrpcClientInstrumentation()
         .AddHttpClientInstrumentation()
         .AddOtlpExporter())
     .WithMetrics(meterBuilder => meterBuilder
+        .AddMeter("OpenTelemetry.Demo.Cart.Meter")
+        .AddView(
+            instrumentName: "CartLatency",
+            new ExplicitBucketHistogramConfiguration {
+                Boundaries = new double[] {300000, 400000, 500000, 600000, 700000, 800000, 900000}
+            }
+        )
```

Comment:
@Kielek, sorry for keeping pinging you 😅 

I'm now trying to set some custom buckets here, but for some reason this is not reflected in Grafana. Any tips?
![image](https://github.com/user-attachments/assets/fad66e4d-1b4d-41a5-a476-e7b9a1ba537e)

Reply:
.NET9/System.Diagnsostic,DiagnosticSource 9.0.0+ (also for .NET8) brings this
https://github.com/dotnet/runtime/issues/63650 and https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.metrics.instrumentadvice-1?view=net-9.0

It can be used like this.

```csharp
    private static readonly Histogram<long> addItemHistogram = CartMeter.CreateHistogram<long>("app.cart.add_item.latency", advice: new InstrumentAdvice<double> { HistogramBucketBoundaries = [1.0, 2.0, 3.0, 4.0], });
    private static readonly Histogram<long> getCartHistogram = CartMeter.CreateHistogram<long>("app.cart.get_cart.latency", advice: new InstrumentAdvice<double> { HistogramBucketBoundaries = [1.0, 2.0, 3.0, 4.0], });
```

You should be able to remove AddView.

Tested with https://github.com/open-telemetry/opentelemetry-dotnet/blob/a6582704cd0701054f4c4cfd6d16f6925fb79144/docs/metrics/exemplars/Program.cs#L20-L25 and OTLP exporter switched to Console exporter.

Reply:
@Kielek you are the best! 🤩 
Thank you!